setup.py

```diff
@@ -50,6 +50,7 @@ def _package_doc(name):
           'Products.PluginRegistry',
           'Products.GenericSetup',
           'Products.StandardCacheManagers',
+          'ZServer',
```

Comment:
Why are we pulling in `ZSevrver`?  For WebDAV stuff?  If so, we should make it an optional dependency (i.e., an "extra" called `webdav`) and ensure that imports are conditional, tests are skipped, etc. if it is missing.